db/models/weatherbase.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const WeatherBase = db.define('weatherbase', {
+  category: {
+    type: Sequelize.ENUM('Thunderstorm', 'Drizzle', 'Rain', 'Snow', 'Clear', 'Cloudy'),
```

Comment:
How about rain AND thunderstorm happening together; the bases are supposed to be mutually exclusive?